geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/vector/WKBGeometryVector.java

```diff
@@ -0,0 +1,109 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.arrow.vector;
+
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.vector.Float8Vector;
+import org.apache.arrow.vector.VarBinaryVector;
+import org.apache.arrow.vector.complex.AbstractContainerVector;
+import org.apache.arrow.vector.complex.FixedSizeListVector;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.FieldType;
+import org.locationtech.geomesa.arrow.vector.impl.AbstractPointVector;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKBReader;
+import org.locationtech.jts.io.WKBWriter;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Catch-all for storing instances of Geometry as WKB
+ */
+public class WKBGeometryVector implements GeometryVector<Geometry, VarBinaryVector> {
+  private VarBinaryVector vector;
+  private WKBWriter writer = new WKBWriter();
+  private WKBReader reader = new WKBReader();
```

Comment:
i guess we don't have the thread-safe pools on the classpath - but i think that should be ok as vectors aren't really thread-safe anyway.

Reply:
Yeah, is there somewhere we could share the Reader/Writers?  (One per vector is kinda gross, right?)

Reply:
it's probably ok to have 1 per vector - usually the issue is you hit thread-safe problems if you do that. maybe we could make them lazy so that we only instantiate them when needed. i think in a lot of use cases we're just copying buffers around so not actually reading/writing through the WKB classes.

Reply:
Since we are in Java land, would having a static and/or thread local thing going on here make sense?

Reply:
you could just use 'java lazy', i.e. make it null to start and then check if it's null each time and instantiate it. i'd say we shouldn't use static or thread locals... i think it's fine to just leave it as-is, too.

Reply:
Ok, great.